deduplicator/src/deduplicator.rs

```diff
@@ -88,7 +88,7 @@ impl Deduplicator {
                         // issue is raised, it would be necessary to implement a specific way of
                         // handling big packs (for example by computing more accurate hashes in
                         // RAM).
-                        teprint!("Performance danger: skipping pack of length {}", pack.len());
+                        teprintln!("Performance danger: skipping pack of length {}", pack.len());
```

Comment:
Don't you think it would be useful to add details about the addresses/position where this case occurs ?
This could help future investigations.

Reply:
What do you think about this: https://github.com/QwantResearch/addresses-importer/pull/41/commits/cdf1a144a37f8e5b4ea0074d1301aac5c5aed30f?

It feels quite dirty and inefficient to dump everything in `stderr`, but this is scenario we want to avoid anyway, and it will be easier to fetch `stderr` than a random log file.

Reply:
Well, I am not sure to understand when this case could occur.  
The addresses contained in the pack would necessarily be concentrated in a very small geographical area, no ? Is that not enough to print some details about the first address ?

Reply:
I'm also not sure what could happen, but as we are using hashing there is no way to ensure that we won't have issues due to collisions the hash function.

I feel like if we only keep the first address there is still a risk to not catch what issue we have :(

Reply:
I just adapted this to only print a sample of 10 addresses: https://github.com/QwantResearch/addresses-importer/commit/721664ef5e826ffed3411892dd56c8a3b12215cb.